File: max_consecutive_ones/max_consecutive_ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int count = 0;
        int consecutiveCount = nums[0];
        for(int i = 0; i < nums.length; i++) {
            if (nums[i] == 1) {
                count++;

            } else {
                count = 0;
            }
            if (count > consecutiveCount) {
                consecutiveCount = count;
            }

        }
        return consecutiveCount;
    }
}

File: max_consecutive_ones/max_consecutive_ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        count = 0
        highest_count = nums[0]
        for val in nums:
            if val == 1:
                count += 1
            
            else:
                count = 0
                
            if count > highest_count:
                highest_count = count
        
        return highest_count
                
        